client/index.html

```diff
@@ -33,12 +35,21 @@
         thead.appendChild(header);
         tableData[0]['values'].forEach(row => {
             var tr = document.createElement('tr');
-            row.forEach(cell => {
+            row.forEach((cell, index) => {
+                if(index==0){ // delete first column "id"
+                    row.deleteCell(0);
+                }
+                if(index == 3){ // round confidence percentages
+                    cell = Math.round(10*cell)/10;
+                }
+                if(index == 4){ // translate time column
+                    cell = translateTime(cell);
+                }
                 var td = document.createElement('td');
                 td.appendChild(document.createTextNode(cell));
                 tr.append(td);
             });
-            tbody.appendChild(tr);
+            tbody.appendChild(tr); 
```

Comment:
Nit: can you enable trimming whitespace on save in your text editor? (If I remember correctly, you are using [vscode](https://stackoverflow.com/a/30884298), yeah?)

Reply:
yeah, thanks for the tip!